delfin/drivers/dell_emc/vmax/constants.py

```diff
@@ -26,3 +26,144 @@
                  "WriteResponseTime"
                  ]
 VMAX_REST_TARGET_URI_ARRAY_PERF = '/performance/Array/metrics'
+
+VMAX_METRICS = [
```

Comment:
This may not be required! for every new metric support we may need add here as well as as reourcetype const